src/domUpdates.js

```diff
@@ -109,9 +111,31 @@ class DomUpdates {
   greetUser(currentUser) {
     const userName = document.querySelector('.user-name');
     userName.innerHTML = `
-    Welcome ${currentUser.name.split(' ')[0]} ${currentUser.name.split(' ')[1][0]}.`;
+    Welcome ${currentUser.name.split(' ')[0]} ${currentUser.name.split(' ')[1][0]}.
+    `
   }
+//should update the dom
+  //after the user has clicked on the images the message should change
+  displayIngredientFeedback(feedback, id) {
+    console.log(id)
+    let card = document.getElementById(id)
+    let name = document.getElementById(`${id} name`)
+    card.innerHTML = ``;
+    card.insertAdjacentHTML('beforeend', `
+    <button class="back-button">No Thanks</button>
```

Comment:
change the name of this button to make more explicit that no action is required

Reply:
Good call moving these to dedicated buttons that are much more descriptive, the utensils and clicking on the picture was definitely not a good end product.

Reply:
We should work on getting the buttons to be disabled and enabled only after certain points in the users interaction. To avoid the spamming of HTML and breaking our forms like we saw last night.

Reply:
I like the idea!!